crates/ruff_python_formatter/tests/snapshots/black_compatibility@cases__form_feeds.py.snap

```diff
@@ -167,8 +167,9 @@ a = [
 +
  #
 -
-+
- #
+-#
+ 
```

Comment:
Ruff does not support form feed characters.

@konstin I think you were right on the on-site. It's form feed characters :)

Reply:
i thought we would, but maybe we only added this for the linter? https://github.com/astral-sh/ruff/pull/7489